### ISSUE TITLE
Extract common content item saving and publishing apis

### ISSUE THREAD

**Issue by hyzx86** [OrchardCMS/OrchardCore#15459]
Currently, it seems that the most complete management of content items is done through the controller. 

Can we extract the logic into the `ContentManager`  class to simplify the secondary development process?

**Comment by MikeAlhayek** [comment]
I don't think it is needed. But if you were to go down that route, I would not pollute the ContentManager with that logic. You could introduce `ContentService` that would wrap that logic.
